static/js/place/overview.tsx

```diff
@@ -30,24 +32,66 @@ interface OverviewPropType {
   locale: string;
 }
 
-class Overview extends React.Component<OverviewPropType> {
+/**
+ * Holds response of /api/place/ranking, used to populate a ranking table.
+ */
+interface OverviewStateType {
+  rankingData: {
+    label: string[];
+    Population: { name: Record<string, unknown>; label: string }[];
+  };
+}
+class Overview extends React.Component<OverviewPropType, OverviewStateType> {
+  constructor(props: OverviewPropType) {
+    super(props);
+    this.state = {
+      rankingData: { label: [], Population: [] },
+    };
+  }
+
   render(): JSX.Element {
     return (
       <section className="factoid col-12">
         <div className="row">
-          <div className="col-12 col-md-4">
-            <GoogleMap dcid={this.props.dcid}></GoogleMap>
-          </div>
-          <div className="col-12 col-md-8">
-            <Ranking
-              dcid={this.props.dcid}
-              locale={this.props.locale}
-            ></Ranking>
+          <div
+            className={`col-12 ${this.shouldShowRanking() ? "col-md-4" : ""}`}
+          >
+            <div className={this.shouldShowRanking() ? "map-with-margin" : ""}>
+              <GoogleMap dcid={this.props.dcid}></GoogleMap>
+            </div>
           </div>
+          {this.shouldShowRanking() ? (
+            <>
+              <div className="col-12 col-md-8">
+                <Ranking
+                  dcid={this.props.dcid}
+                  locale={this.props.locale}
+                  data={this.state.rankingData}
+                ></Ranking>
+              </div>
+            </>
+          ) : (
+            <></>
```

Comment:
this doesn't need to be an if else check, can just do `shouldShowRanking && <div ...><Ranking .../></div> ` which says only render the ranking div if shouldShowRanking

Reply:
good catch! Switched to using &&